### PR TITLE
feat(aws-ecs): aws-ecs module structure

### PR DESCRIPTION
**Pull request by BrunoCarrier**
Introduced the AWS ECS module to the project

I am not sure what should be the namespace for properties for the ECS cloud provided.  I used "aws.ecs.enabled".  Is that consistent with the vision you had?

@robzienert @cfieber @ajordens 

Also, should I have included more people than the above?

### CODE REVIEW

File: clouddriver-web/clouddriver-web.gradle [hunk 1]
```diff
@@ -22,6 +22,7 @@ repositories {
 
 dependencies {
   compile project(':clouddriver-aws')
+  compile project(':clouddriver-aws-ecs')
```
Comment (tomaslin) [inline clouddriver-web/clouddriver-web.gradle]: make this `clouddriver-ecs` to follow the conventions of clouddriver-appengine and clouddriver-gce

File: clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy [hunk 2]
```diff
@@ -54,6 +55,7 @@ import java.security.Security
   DeployConfiguration,
   RetrofitConfig,
   AwsConfiguration,
+  AwsEcsConfiguration,
```
Comment (tomaslin) [inline clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy]: EcsConfiguration

File: clouddriver-web/clouddriver-web.gradle (hunk 1 above)
Comment (BrunoCarrier) [inline clouddriver-web/clouddriver-web.gradle]: sure, will do

File: clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy (hunk 2 above)
Comment (BrunoCarrier) [inline clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/Main.groovy]: Will do

**Comment by BrunoCarrier** [comment]
@tomaslin I made the requested changes

**Comment by tomaslin** [comment]
LGTM

**Pull request merged**